File: python/get_campaign_statut.py
#replace with your own values

import http.client

conn = http.client.HTTPSConnection("localhost", 3333)
payload = ''
headers = {
  'Authorization': 'Bearer dc408be00542eed0f4e7d1336d9a38735e5fa74'
}
conn.request("GET", "/v1/campaigns/1/status", payload, headers)
res = conn.getresponse()
data = res.read()
print(data.decode("utf-8"))

File: php/get_campaign_statut.php
<?php
$client = new Client();
$headers = [
  'Authorization' => 'Bearer dc408be00542eed70f4e7d1336d9a38735e5fa74'
];
$body = '';
$request = new Request('GET', 'http://localhost:3333/v1/campaigns/1/status', $headers, $body);
$res = $client->sendAsync($request)->wait();
echo $res->getBody();
